test.js

```diff
@@ -60,6 +60,7 @@ test('list - custom indicator', t => {
 
 	const CustomIndicator = () => 'X ';
 
+        // eslint-disable-next-line  react/jsx-no-bind
```

Comment:
These eslint comments aren't aligned with the code and I don't think they are necessary. There's no `bind()` on the next line.